src/providers/twitch/twitchapi.cpp

```diff
@@ -0,0 +1,37 @@
+#include "providers/twitch/twitchapi.hpp"
+
+#include "Application.hpp"
+#include "common/UrlFetch.hpp"
+#include "controllers/accounts/AccountController.hpp"
+#include "providers/twitch/TwitchCommon.hpp"
+
+#include <QString>
+#include <QThread>
+
+namespace chatterino {
+
+void TwitchApi::FindUserId(const QString user, std::function<void(QString)> callback)
+{
+    QString requestUrl("https://api.twitch.tv/kraken/users?login=" + user +
+                       "&api_version=5&client_id=" + getDefaultClientID());
```

Comment:
Don't use api_version and client_id in the url. Use `request.makeAuthorizedV5(getDefaultClientID());`